ext_mongodb.php

```diff
@@ -200,7 +200,7 @@ function($value) {
 		if ($this->writeConcern) {
 			$ret['writeConcern'] = $this->writeConcern;
 		} else {
-			$ret['writeConcern'] = NULL;
+			$ret['writeConcern'] = [];
```

Comment:
I wonder if `null` is a better choice here, as that's what we're doing with the `writeConcernError` field. I'd be willing to change PHPC instead.